.github/workflows/helm_publish.yaml

```diff
@@ -5,7 +5,8 @@ on:
     branches:
       - main
     paths:
-      - helm/fuel-streams-publisher/Chart.yaml
+      - cluster/charts/fuel-streams-publisher/Chart.yaml
+      - cluster/charts/fuel-nats/Chart.yaml
```

Comment:
This is the new NATS helm charts with out base configuration, so we can just use it then in production